src/utils/template_handler.rs

```diff
@@ -200,6 +259,52 @@ pub(crate) fn load_remote_template(path: &str, url: &str, force: bool) -> Status
         }
     }
 
+    return Status::ok();
+}
+
+/// Load a template from a remote repository
+pub(crate) fn load_remote_template(
```

Comment:
Move that function above `load_gitlab_template` and `load_github_template`because this function should be used and therefore should be seen first

Reply:
For sure, changed the position of function, for better readablity and maintainablity as you suggested